chart/hyrax/templates/_tmplvalues.tpl

```diff
@@ -0,0 +1,38 @@
+{{/*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Renders a value that contains template perhaps with scope if the scope is present.
+Usage:
+{{ include "common.tplvalues.render" ( dict "value" .Values.path.to.the.Value "context" $ ) }}
```

Comment:
we don't want to add a dependency on the bitnami `common` chart, do we?

Reply:
oh i guess it's just re-using the name; in any case let's use `hyrax` instead

Reply:
I don't see any references to Bitnami other than the copyright notice. The file is a copy of the upstream, but since it is Apache 2, as is Hyrax, the two are compatible license wise. I feel like leaving the attribution is a good thing here, but if you are concerned about it, I'll do some mods and then recommit it.

Reply:
wow - I can't read apparently. I see what you mean now. I'll fix it